src/screens/User/User.tsx

```diff
@@ -1,37 +1,51 @@
-import React from 'react'
```

Comment:
Maybe it will be better if name this file and component as `Profile`?

Reply:
agreed

Reply:
Let's do it.